src/aeso_fcode_to_fate.erl

```diff
@@ -1094,10 +1089,14 @@ simpl_s(I, _) -> I.
 simpl_top(I, Code, Options) ->
     simpl_top(?SIMPL_FUEL, I, Code, Options).
 
+%% Change r_rule to optimize_rule (e.g. change r_push_consume to optimize_push_consume)
+-define(RULE_OPTIMIZATION(Rule), list_to_atom("optimize" ++ lists:nthtail(1, atom_to_list(Rule)))).
```

Comment:
`list_to_existing_atom` is strongly preferred -- did you check if it is not an option?

Reply:
I don't think that `list_to_existing_atom/1` would work here. I am creating the atoms for the first time in this macro, they do not exist anywhere else.